app/models/manageiq/providers/ansible_tower/configuration_manager/playbook.rb

```diff
@@ -0,0 +1,4 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::Playbook < ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript
+  belongs_to  :configuration_project, class_name: "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationProject"
+  has_many    :configuration_script,  class_name: "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript"
```

Comment:
I would expect the relations to live in the base ConfigurationScript class and not be specific to Ansible at all.